src/iterable-functions.php

```diff
@@ -114,6 +114,20 @@ function iterable_reduce(iterable $iterable, callable $reduce, $initial = null)
     return $initial;
 }
 
+/**
+ * Yields iterable values (leaving out keys).
+ *
+ * @param iterable<TValue> $iterable
+ *
+ * @return iterable<int, TValue>
+ *
+ * @template TValue
+ */
+function iterable_values(iterable $iterable): iterable
+{
+    return iterable($iterable)->values();
```

Comment:
Could you make a similar return as [here](https://github.com/bpolaszek/php-iterable-functions/blob/cfceadb56a54ef93076ee80c5028668ed93327f8/src/iterable-functions.php#L31) to be consistent? In functions context, I prefer returning the same iterable primary type as the input.

Reply:
@bpolaszek BTW I slightly think the approach is not ideal because now chaining functions does not leverage generator benefits.

```php
compose(
    'iterable_values',
    'iterable_map',
    'iterable_filter'
)($array)
```

With array I think the space complexity is 3 times higher than with Traversable because array gets rebuilt 3 times.